src/models/analytics/analytics-tables.ts

```diff
@@ -525,6 +529,24 @@ const analyticsTablesL10n: Record<TableKeys, AnalyticsTable> = {
       nl: { singular: 'Issue', plural: 'Issues' },
     },
   },
+  issueAssignedUser: {
+    fields: ['hashId', 'name'],
+    fieldsWithTranslations: {
+      en: {
+        hashId: { singular: 'ID', plural: 'IDs' },
+        name: { singular: 'Name', plural: 'Names' },
+      },
+      nl: {
+        hashId: { singular: 'ID', plural: 'IDs' },
+        name: { singular: 'Naam', plural: 'Namen' },
+      },
+    },
+    tableKey: 'issueAssignedUser',
+    tableText: {
+      en: { singular: 'Issue assigned user', plural: 'Issue assigned users' },
+      nl: { singular: 'Issue toegewezen aan gebruiker', plural: 'Issue toegewezen aan gebruikers' },
```

Comment:
komt dit mooi uit op je scherm? Ik heb moeite een kortere suggestie te geven maar 'toegewezen aan' zou met de context van de tabel erbij ook al kunnen werken. of 'Issue toegewezen aan'